src/internal/m365/collection/drive/helper_test.go

```diff
@@ -778,40 +785,46 @@ func aReset(items ...models.DriveItemable) mock.NextPage {
 // delta trees
 // ---------------------------------------------------------------------------
 
-var loc = path.NewElements("root:/foo/bar/baz/qux/fnords/smarf/voi/zumba/bangles/howdyhowdyhowdy")
+func defaultLoc() path.Elements {
+	return path.NewElements("root:/foo/bar/baz/qux/fnords/smarf/voi/zumba/bangles/howdyhowdyhowdy")
+}
+
+func newTree(t *testing.T) *folderyMcFolderFace {
+	return newFolderyMcFolderFace(fullPathPath(t), rootID)
+}
 
-func treeWithRoot() *folderyMcFolderFace {
-	tree := newFolderyMcFolderFace(nil, rootID)
+func treeWithRoot(t *testing.T) *folderyMcFolderFace {
+	tree := newFolderyMcFolderFace(fullPathPath(t), rootID)
 	rootey := newNodeyMcNodeFace(nil, rootID, rootName, false)
```

Comment:
I'm not sure if you have another PR to update these calls as I haven't looked at them all yet, but now that we're passing `t` in could you please switch calls that add things to the tree to use the existing tree manipulation functions where possible?

Reply:
it's a ways out, but I have that change here: https://github.com/alcionai/corso/pull/4821/files#diff-48bc25f85f8c885368934862a2813c6fbd7dfb71f309f91169f605230a85895eR463